### ISSUE TITLE
Azure data studio slow to open windows 

### ISSUE THREAD

**Issue by jmetape** [microsoft/azuredatastudio#21747]
- Azure Data Studio Version: 
- - Version: 1.41.0
Commit: cf5e0fe1a33709e2a82dc94f42dbe88487803d62
Date: 2023-01-24T17:29:04.297Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0

- OS Version: Darwin x64 22.2.0 / MacOS Ventura 13.1

Steps to Reproduce:

1. Open azure data studio, and notice how long it takes to load the window. Open a workspace or folder -- very slow. 
2. Open a couple more windows, and open a folder / workspace, the 3rd open window doesn't open the folder / workspace, it just displays a white screen 

Does this issue occur when all extensions are disabled?:  Yes

**Comment by alanrenmsft** [comment]
@jmetape does it help if you disable the telemetry? you can do it by changing the setting in screenshot below to `off`.

![image](https://user-images.githubusercontent.com/13777222/215153886-aa08286e-5d37-4899-860c-911be7831528.png)

**Comment by jmetape** [comment]
No, It seemed to have made it worse, I can have one window open, the 2nd one just sits at a white screen

**Comment by alanrenmsft** [comment]
@jmetape have you tried rolling back to the previous version? I want to rule out the machine issue.

**Comment by jmetape** [comment]
I rolled it back to this version:
<img width="265" alt="ads_version" src="https://user-images.githubusercontent.com/10228011/215225753-3019e6ea-d24a-433f-a5dc-015b541eece7.png">


The windows load much faster, but the 3rd window still fails to load

**Comment by jmetape** [comment]
I discovered a "workaround"  simply by chance where I can open multiple windows.  I've successfully opened 5 windows and loaded workspaces into them. This is how I did it, I realize this might be difficult to follow. 
Lets say I have 5 workspaces:

WorkspaceA.code-workspace
WorkspaceB.code-workspace
WorkspaceC.code-workspace
WorkspaceD.code-workspace
WorkspaceE.code-workspace

With no windows open, I open a new window, open WorkSpaceA-it opens. Open a second window, open WorkSpaceB-it opens.  Open a third window, open WorkspaceC, it fails to load--just a white screen. I close that window, go to WorkspaceB window and open WorkspaceC in that window. I open a new window and open WorkspaceB, it opens. I go to Workspace C window, open WorkspaceD in that window --it opens. I open a new window and open WorkspaceC --it opens. Now I have 4 windows open. In the WorkspaceD window I open Workspace E -- it opens. I open a fifth window and open WorkspaceD --it opens.
